src/main/scala/com/updateimpact/Plugin.scala

```diff
@@ -173,4 +173,38 @@ object Plugin extends AutoPlugin {
       desktop.browse(URI.create(url))
     }
   }
+
+  //Swaps internal module dependencies to includes it's dependencies.
+  //If module A depends on local module B we only get info about B without it's dependencies.
+  //Example:
+  //  A depends on cats and module B.
+  //  B depends on macwire.
+  //
+  //  A's tree before (there is no macwire):
+  //    A_
+  //      |_cats
+  //      |_B
+  //
+  //  A's Tree after:
+  //    A_
+  //      |_cats
+  //      |_B
+  //        |_macwire
+  def modulesWithFullLocalDependenciesInfo(modules: Seq[ModuleDependencies]): Seq[ModuleDependencies] = {
+    def extractRootDependency(module: ModuleDependencies) = module
+      .getDependencies.asScala
+      .filter(_.getId == module.getModuleId)
+      .head
+
+    modules.groupBy(_.getConfig).flatMap { case (config, modulesWithinConfig) =>
+      val modulesRootDependencies = modulesWithinConfig.map(extractRootDependency)
+      modulesWithinConfig.map { module =>
+        val dependencyWithFullInfo = module.getDependencies.asScala.map { dependency =>
+          val dependencyWithFullInfo = modulesRootDependencies.find(_.getId == dependency.getId)
+          dependencyWithFullInfo.getOrElse(dependency)
```

Comment:
will transitive dependencies (if B depens on lib1 which depends on lib2) be included as well?